platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/VirtualMapBaseBench.java

```diff
@@ -167,6 +196,7 @@ protected VirtualMap<BenchmarkKey, BenchmarkValue> copyMap(
      */
     protected VirtualMap<BenchmarkKey, BenchmarkValue> flushMap(
             final VirtualMap<BenchmarkKey, BenchmarkValue> virtualMap) {
+        logger.info("Flushing map {}...", virtualMap.getLabel());
```

Comment:
Debug output?

Reply:
Similar to above, this extra output helps track the progress of the benchmark. Also, this one is simply improving the already existing debug output by adding an opening log in addition to the closing one. This way, if the operation freezes, one knows what was going on just before.